upgrade_tests/upgrade_udtfix_test.py

```diff
@@ -154,7 +155,7 @@ def _verify_upgrade_log(self, log_mark, node):
 
     start_family = spec['UPGRADE_PATH'].starting_meta.family
     upgrade_family = spec['UPGRADE_PATH'].upgrade_meta.family
-    start_family_applies = start_family == '3.0' and (upgrade_family == 'trunk' or LooseVersion(upgrade_family) > '3.0')
+    start_family_applies = start_family == CASSANDRA_3_0
```

Comment:
should we not keep the `upgrade_family > CASSANDRA_3_0` condition?

Reply:
`start_family_applies`... 
However the test is tagged with `@since('3.11.6')`